library/core/src/hint.rs

```diff
@@ -287,7 +287,7 @@ pub fn spin_loop() {
 /// - Treats the call to `contains` and its result as volatile: the body of `benchmark` cannot
 ///   optimize this away
 ///
-/// This makes our benchmark much more realistic to how the function would be used in situ, where
+/// This makes our benchmark much more realistic to how the function would be used in situations where
```

Comment:
@4gboframram Would you be willing to make the change suggested by @danielhenrymantilla?
```suggestion
/// This makes our benchmark much more realistic to how the function would actually be used, where
```